ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh

```diff
@@ -7,6 +7,52 @@ set -o nounset
 IBMCLOUD_HOME_FOLDER=/tmp/ibmcloud
 NO_OF_RETRY=${NO_OF_RETRY:-"5"}
 
+############################################################
+# Set PowerVS and VPC Zone and Region  
+POWERVS_ZONE="${LEASED_RESOURCE}"
+POWERVS_REGION=$(
```

Comment:
this is OK, I wanted to contain it a bit. any reasoning behind moving back? maybe we should update a comment on L11 (expand on what is there).?

Reply:
The reason for moving to local variables because the VPC and PVC region are been used even before the configure , as we use the IBM login .so I had moved to global so these export variables been used be other functions .

Reply:
sounds good